packages/api-client/src/views/Request/libs/oauth2.test.ts

```diff
@@ -147,14 +150,98 @@ describe('oauth2', () => {
       })
     })
 
+    // PKCE
+    it('should generate valid PKCE code verifier and challenge', async () => {
+      const _scheme = {
+        ...scheme,
+        flow: {
+          ...scheme.flow,
+          'x-usePkce': true,
+        },
+      }
+      const accessToken = 'pkce_access_token_123'
+      const codeChallenge = 'AQIDBAUGCAkK'
+      const code = 'pkce_auth_code_123'
+
+      // Mock crypto.getRandomValues
+      vi.spyOn(crypto, 'getRandomValues').mockImplementation((arr) => {
+        if (arr instanceof Uint8Array) {
+          for (let i = 0; i < arr.length; i++) {
+            arr[i] = i
+          }
+        }
+        return arr
+      })
+
+      // Mock crypto.subtle.digest
+      vi.spyOn(crypto.subtle, 'digest').mockResolvedValue(
+        new Uint8Array([1, 2, 3, 4, 5, 6, 8, 9, 10]).buffer,
```

Comment:
This should be 32 bytes.

Reply:
Would it matter if its just a mock? It really doesn't do much

Reply:
Nvm got it will change the generator